test/sdk/cpp/server.cc

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A server that starts, and then stops after 60 seconds.
+// This example really does nothing other than show how to integrate
+// the C++ SDK.
+
+#include <agones/sdk.h>
+#include <grpc++/grpc++.h>
+#include <iostream>
+#include <thread>
+
+// send health check pings
+void DoHealth(std::shared_ptr<agones::SDK> sdk) {
```

Comment:
Is this file meaningfully different form examples/cpp-simple/server.cc? On quick glance, it looks pretty similar, except that #1283 is fixing an issue with thread joining that this test server doesn't have.

Reply:
Yes, this file is different in that it performs all SDK function invocations, in order to pass the conformance test. So it should not be treated as an example of how to write gameservers using CPP SDK.